cypress/e2e/cloud/org-list.test.ts

```diff
@@ -68,9 +68,7 @@ describe('Account / Organizations Tab', () => {
     )
 
     cy.getByTestID('question-mark-tooltip--tooltip--dialog').within(() => {
-      cy.get('a')
-        .contains('https://support.influxdata.com/s/login')
-        .should('have.attr', 'href', 'https://support.influxdata.com/s/login')
+      cy.get('a').contains('contact support').should('have.attr', 'href', '#')
```

Comment:
See later comments -- We should update this test to check the onClick behavior, not for the presence of the href attribute.